### ISSUE TITLE
Workspaces revamp

### ISSUE THREAD

**Issue by leandrotruppa** [getferdi/ferdi#1753]
Hey guys,

First of all i want to apologize for my poor english. I am using Rambox Pro for a year now and discover Ferdi a few days ago. 

I find something anoying in Ferdi about the Workspaces: If you are using a couple of Workspaces you have to mantain open a BIG sidebar to switch between workspaces. Maybe you can change the look of the sidebar to looks like this:

![image](https://user-images.githubusercontent.com/8252337/128607099-6fb74105-5275-46cc-8dda-d06dbb04c953.png)

Also here is a screenshot of the create workspace area:

![image](https://user-images.githubusercontent.com/8252337/128607151-6288d7a5-2a1f-4252-a738-9074a5f12680.png)

**Comment by vraravam** [comment]
Nice suggestion @leandrotruppa - we will see if we can prioritize this soon.

**Comment by CoinAxe** [comment]
Good idea but please don't make this the default, I'm already using the buttons on my streamdeck to switch between workspaces and this only adds more unnecessary width. If the OP means this would only show when the workspace drawer shows, then yea I agree with him.

**Comment by leandrotruppa** [comment]
> Good idea but please don't make this the default, I'm already using the buttons on my streamdeck to switch between workspaces and this only adds more unnecessary width. If the OP means this would only show when the workspace drawer shows, then yea I agree with him.

The idea is modify the way its shown when you select to open workspaces or when you select to "Always show Workspaces sidebar"

**Comment by SkepticSpoon** [comment]
Hello! Came here to suggest this! I'd like to have the workspaces sidebar always open to quickly switch between workspaces, but it's too wide on my ultrabook monitor. A way to have it only be icons (which we could choose for each workspace) or even initials or something like that would be great
